UI/api-interface.cpp

```diff
@@ -626,6 +626,19 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 			cb.callback(event, cb.private_data);
 		}
 	}
+
+	void obs_frontend_multitrack_video_register(const char *name,
```

Comment:
its only purpose is to enable views to participate in multitrack streaming, at least I'm not aware of any other functionality that would make use of this right now

upcoming webrtc multitrack (and other/future multitrack outputs) might be interested in this at some point, but so far webrtc multitrack is stuck in review

doing this in libobs-land is somewhat very uncharted territory, since views/video outputs/general obs objects don't carry meta information about their intended purpose (and given the private data capabilities, they're currently very much in "don't touch these objects unless you know where they come from/why they're there" territory)